frontend/javascripts/oxalis/view/settings/dataset_settings_view.js

```diff
@@ -329,6 +351,14 @@ class DatasetSettings extends React.PureComponent<DatasetSettingsProps, State> {
     );
   };
 
+  reloadWholeDatasetData = async (layerName: string): Promise<void> => {
+    await clearCache(this.props.dataset);
+    api.data.reloadBuckets(layerName);
+    Toast.success(
+      `Successfully deleted cached data of layer ${layerName}. Now move a bit around to reload the data.`,
```

Comment:
We shouldn't force the user to move around to see the new data. Setting `window.needsRerender = true;` above should do the trick here :)

Reply:
This does not work for me. Do you have any other suggestions or know why it might not be working?

Reply:
Hm, let's have a look together at this.

Reply:
maybe this helps `app.vent.trigger("rerender");` :man_shrugging: . Just saw it in our code base.

Reply:
I had a look and fixed it now :) The layer rendering manager had to be refreshed.